engine/dkg/processor/engine.go

```diff
@@ -123,14 +126,14 @@ func (e *Engine) Process(originID flow.Identifier, event interface{}) error {
 
 func (e *Engine) process(originID flow.Identifier, event interface{}) error {
 	switch v := event.(type) {
-	case dkg.DKGMessage:
-		return e.onDKGMessage(originID, v)
+	case msg.DKGMessage:
+		return e.onMessage(originID, v)
```

Comment:
this question is rather related to the previous PR: In the case of a 1-1 message, where does the parameter `originID` come from? is it set by the lower networking layer?

Reply:
yes it is set by the lower networking layer

Reply:
That's great!